ee/clickhouse/sql/session_recording_events.py

```diff
@@ -63,9 +70,11 @@
 
 
 INSERT_SESSION_RECORDING_EVENT_SQL = """
-INSERT INTO session_recording_events SELECT %(uuid)s, %(timestamp)s, %(team_id)s, %(distinct_id)s, %(session_id)s, %(window_id)s, %(snapshot_data)s, %(created_at)s, now(), 0
+INSERT INTO session_recording_events (uuid, timestamp, team_id, distinct_id, session_id, window_id, snapshot_data, created_at, _timestamp, _offset)
```

Comment:
Updating this insert statement to specify columns because of the comment in our [blog post](https://posthog.com/blog/clickhouse-materialized-columns#backfilling-efficiently).